ABOUT.md

```diff
@@ -1,9 +1,106 @@
+#### Shogun is accessible
 
-The wealth of ML open-source software allows us to offer bindings to other sophisticated libraries including: [LibSVM](http://www.csie.ntu.edu.tw/~cjlin/libsvm/)/[LibLinear](http://www.csie.ntu.edu.tw/~cjlin/liblinear/), [SVMLight](http://svmlight.joachims.org/), [LibOCAS](http://cmp.felk.cvut.cz/~xfrancv/ocas/html/), [libqp](http://cmp.felk.cvut.cz/~xfrancv/libqp/html/), [VowpalWabbit](http://www.hunch.net/~vw/), [Tapkee](http://tapkee.lisitsyn.me/), [SLEP](http://www.public.asu.edu/~jye02/Software/SLEP/), [GPML](http://www.gaussianprocess.org/gpml/code/matlab/doc/) and more. See our [list of integrated external libraries](http://www.shogun-toolbox.org/page/about/contributions).
+* Supports *many languages* (Python, Octave, R, Java/Scala, Lua, C#, Ruby, etc) and *platforms* (Linux/Unix, MacOS and Windows), easy integration with other ML libraries
+
+#### Shogun is state-of-the-art
+
+* *efficient and competitive* implementation (from standard to cutting edge algorithms), modern software architecture in C++
+* easy combination of *multiple data representations, algorithm classes and general purpose tools* for rapid prototyping of data pipelines
+
+#### Shogun is open source
+
+* *free* and *community-based* machine learning education (see [Using Shogun to Learn](#useshoguntolearn))
+* *GPLv3 License* and working towards BSD compatibility
+
+You can try Shogun in our [cloud](https://cloud.shogun.ml) (no installation required) or find out more about our [mission](#mission), [features](#features) and [history](#shogunhistory).
+
+
+
+
+
+
+## <a name="mission"></a> Mission
+
+Machine learning is powerful. 
+Within the last two decades, machine learning (ML) has emerged from an eccentric academic field to transforming the way we live our everyday life. 
+Its methods influence society and its tools have a billion-dollar impact on global economy. 
+We want to make sure that the underlying machine learning algorithms are transparent and accessible. 
+We believe that ML should not be a privilege of corporations or political power with the money to buy technical knowledge. 
+Anyone - scientists, data enthusiasts, non-profit organisations, hackers, citizens, journalists, students - should be able to use ML tools for free.
+
+The Shogun ML toolbox promotes this mission through its features (*accessible*, *open source* and *good*) and a focus on ML education and developing its contributors.
+
+We are proud to be a big and very active open source ML platform.
```

Comment:
@karlnapf suggested to include this. How about we change it to more of an aim by saying "we are constantly improving Shogun (implies "very active") and aim to grow the platform"?